projects/transactions-journey/src/lib/communication/index.ts

```diff
@@ -0,0 +1,11 @@
+import { InjectionToken } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Transaction } from '../model/transaction';
+
+export interface TransactionsCommunicationService {
+  latestTransaction$: Observable<Transaction | undefined>;
+}
+
+export const TRANSACTIONS_JOURNEY_COMMUNICATION_SERIVCE = new InjectionToken<TransactionsCommunicationService>(
+  'bb-transactions-journey communication service',
+);
```

Comment:
Why we use `InjectionToken` instead of `@Injectable` here? Is it just for demonstration?

Reply:
Because the service is needed to be created on the application level and the journey is expected to receive it from the upper injector.

Reply:
You can achieve the same result with an `@Injectable`.

Reply:
Agreed in slack, that this is okay